runtime/jcl/common/criu.cpp

```diff
@@ -38,6 +38,14 @@ Java_openj9_internal_criu_InternalCRIUSupport_getCheckpointRestoreNanoTimeDeltaI
 	return currentThread->javaVM->checkpointState.checkpointRestoreTimeDelta;
 }
 
+jlong JNICALL
+Java_openj9_internal_criu_InternalCRIUSupport_getLastRestoreTimeImpl(JNIEnv *env, jclass unused)
+{
+	J9VMThread *currentThread = (J9VMThread *)env;
+
+	return currentThread->javaVM->checkpointState.lastRestoreTimeMillis;
```

Comment:
Where does this get the value `-1` as documented for `getLastRestoreTime()`?

Reply:
Yeah, fixed it at https://github.com/eclipse-openj9/openj9/pull/18235/commits/aa2b96f8f746a9635f027400fc2d1d73c1959cbd

Reply:
Please mention the PR here when you create one with that fix.

Reply:
The PR containing the fix
* https://github.com/eclipse-openj9/openj9/pull/18235